sfa_dash/blueprints/main.py

```diff
@@ -69,6 +197,15 @@ def get_breadcrumb_dict(self):
             breadcrumb_dict[f'{self.human_label}s'] = url_for(
                 f'data_dashboard.{listing_view}',
                 uuid=self.metadata['site_id'])
+        elif self.metadata.get('aggregate') is not None:
```

Comment:
I don't think aggregates logically belong with forecasts. They are more like observations without sites.

Reply:
I think having the breadcrumb relating the forecast to the Aggregate is helpful. That relationship is duplicated in the metadata section as well though, so I'm not completely attached to it.

Reply:
whoops. I didn't look carefully and thought this was for data_type == 'aggregate' in the __init__. I don't have a problem with keeping this breadcrumb